### PR TITLE
Code blocks UI with escaped html

### PR DESCRIPTION
**Pull request by rlisahuang**

### CODE REVIEW

**Comment by NikolajBjorner** [comment]
Is Peli supposed to sign of?

**Comment by rlisahuang** [comment]
> 

Yes, I let him to do a final review

**Comment by NikolajBjorner** [comment]
alright

**Comment by rlisahuang** [comment]
@pelikhan the last commit escapes quotation marks of all sorts, which was a bug in the commit before it

**Comment by pelikhan** [comment]
Looks great, ready to merge

**Review by pelikhan — APPROVED** [review]

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -4,121 +4,129 @@
...
+        '@docusaurus/preset-classic',
+        // /** @type {import('@docusaurus/preset-classic').Options} */
+        ({
+          docs: {
+            remarkPlugins: [],
+            sidebarPath: require.resolve('./sidebars.js'),
+            // Please change this to your repo.
+            // Remove this to remove the "edit this page" links.
+            editUrl:
+              'https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/',
```
Comment (pelikhan) [inline website/docusaurus.config.js:46]: this edit link need to be updated

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -4,121 +4,129 @@
...
+                  href: 'https://twitter.com/docusaurus',
+                },
+              ],
+            },
+            {
+              title: 'More',
+              items: [
+                {
+                  label: 'GitHub',
+                  href: 'https://github.com/facebook/docusaurus',
```
Comment (NikolajBjorner) [inline website/docusaurus.config.js:113]: there are a lot of links to docusaurus on this site. @pelikhan - shouldn't it be overhauled to RiSE / MSR information?

**Pull request merged**